### PR TITLE
Add WITH_SSB_WEB to README

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Many board that run [Armbian](https://www.armbian.com/) such as many models of O
     The installation script can also install many optional features such as distributed applications and network analysis tools that are useful but non-essential to run a node. You can use flags to selectively enable them, or use the following command to install all optional features:
 
     ```
-    $ wget https://raw.githubusercontent.com/tomeshnet/prototype-cjdns-pi/master/scripts/install && chmod +x install && WITH_MESH_POINT=true WITH_AD_HOC=false WITH_WIFI_AP=true WITH_FIREWALL=true WITH_CJDNS_IPTUNNEL=true WITH_IPFS=true WITH_SSB=true WITH_PROMETHEUS_NODE_EXPORTER=true WITH_PROMETHEUS_SERVER=true WITH_GRAFANA=true WITH_H_DNS=true WITH_H_NTP=true WITH_EXTRA_TOOLS=true ./install
+    $ wget https://raw.githubusercontent.com/tomeshnet/prototype-cjdns-pi/master/scripts/install && chmod +x install && WITH_MESH_POINT=true WITH_AD_HOC=false WITH_WIFI_AP=true WITH_FIREWALL=true WITH_CJDNS_IPTUNNEL=true WITH_IPFS=true WITH_SSB=true WITH_SSB_WEB=true WITH_PROMETHEUS_NODE_EXPORTER=true WITH_PROMETHEUS_SERVER=true WITH_GRAFANA=true WITH_H_DNS=true WITH_H_NTP=true WITH_EXTRA_TOOLS=true ./install
```
Comment (darkdrgn2k) [inline README.md]: This line is getting pretty big. We sure we want to include everything

I didn't include pi-stream due to privacy concerns (it streaming automatically)

Comment (benhylau) [reply README.md]: Yea this is kinda unmanageable I don't even use this now, bc never do I wanna actually install everything... include default profiles in the script itself and just name the profile in the option?

Comment (benhylau) [reply README.md]: `PROFILE=core|default|all|meshstream|experimental`

Comment (darkdrgn2k) [reply README.md]: In fact if you WHERE to install everything your pi would probably start crying!

**Pull request merged**